2.2 and related updates (#451)
atbcb/ICTTestingBaseline: _baselines/21TimedEvents.md

-   [Conformance Requirement 5: Non-Interference](https://www.w3.org/WAI/WCAG22/Understanding/conformance#conf-req5) - The following success criteria apply to all content on the page, including content that is not otherwise relied upon to meet conformance, because failure to meet them could interfere with any use of the page: 1.4.2 - Audio Control, 2.1.2 - No Keyboard Trap, 2.3.1 - Three Flashes or Below Threshold, and 2.2.2 - Pause, Stop, Hide.

### Test Method Rationale

Determine how time limits, auto-play, and auto-update can be modified by a user and execute the modifications.

### Limitations, Assumptions, or Exceptions

-   From [SC 2.2.1: Timing Adjustable](https://www.w3.org/WAI/WCAG22/Understanding/timing-adjustable), time limits set by the content that meet any of the following are not included in this test:
    -   **Real-time Exception:** The time limit is a required part of a real-time event (for example, an auction), and no alternative to the time limit is possible; or
    -   **Essential Exception:** The time limit is essential and extending it would invalidate the activity; or
    -   **20 Hour Exception:** The time limit is longer than 20 hours.

>     -   Also, content that repeats or is synchronized with other content, so long as the information and data is adjustable or otherwise under the control of the end user. Examples of time limits for which this success criterion is not applicable include scrolling text that repeats, captioning, and carousels. These are situations which do include time limits, but the content is still available to the user because it has controls for accessing it

Author: ```suggestion
    -   Content that repeats or is synchronized with other content, so long as the information and data is adjustable or otherwise under the control of the end user. Examples of time limits for which this success criterion is not applicable include scrolling text that repeats, captioning, and carousels. These are situations which do include time limits, but the content is still available to the user because there are controls for accessing it.
```

### Limitations, Assumptions, or Exceptions

-   From [SC 2.2.1: Timing Adjustable](https://www.w3.org/WAI/WCAG22/Understanding/timing-adjustable), time limits set by the content that meet any of the following are not included in this test:
    -   **Real-time Exception:** The time limit is a required part of a real-time event (for example, an auction), and no alternative to the time limit is possible; or
    -   **Essential Exception:** The time limit is essential and extending it would invalidate the activity; or
    -   **20 Hour Exception:** The time limit is longer than 20 hours.

>     -   Also, content that repeats or is synchronized with other content, so long as the information and data is adjustable or otherwise under the control of the end user. Examples of time limits for which this success criterion is not applicable include scrolling text that repeats, captioning, and carousels. These are situations which do include time limits, but the content is still available to the user because it has controls for accessing it

Reviewer: A couple editorial suggestions:

- Remove "Also" at the beginning. 
- Add period at the end of the sentence. 
- swap "because it has controls for accessing it" to "because controls are available for accessing it."

Content that repeats or is synchronized with other content, so long as the information and data is adjustable or otherwise under the control of the end user. Examples of time limits for which this success criterion is not applicable include scrolling text that repeats, captioning, and carousels. These are situations which do include time limits, but the content is still available to the user because controls are available for accessing it.
